--- a/newlib/libc/include/sys/stat.h    2022-10-23 15:34:20.636665821 +0200
+++ b/newlib/libc/include/sys/stat.h    2022-10-23 16:29:26.606666652 +0200
@@ -34,27 +34,16 @@
   gid_t                st_gid;
   dev_t                st_rdev;
   off_t                st_size;
-#if defined(__svr4__) && !defined(__PPC__) && !defined(__sun__)
-  time_t       st_atime;
-  time_t       st_mtime;
-  time_t       st_ctime;
-#else
   struct timespec st_atim;
   struct timespec st_mtim;
   struct timespec st_ctim;
   blksize_t     st_blksize;
   blkcnt_t     st_blocks;
-#if !defined(__rtems__)
-  long         st_spare4[2];
-#endif
-#endif
 };
 
-#if !(defined(__svr4__) && !defined(__PPC__) && !defined(__sun__)) && !defined(__cris__)
 #define st_atime st_atim.tv_sec
 #define st_ctime st_ctim.tv_sec
 #define st_mtime st_mtim.tv_sec
-#endif
 
 #endif
 
