--- a/newlib/libc/include/sys/stat.h
+++ b/newlib/libc/include/sys/stat.h
@@ -142,7 +142,7 @@
 int    stat (const char *__restrict __path, struct stat *__restrict __sbuf );
 mode_t umask (mode_t __mask );
 
-#if defined (__SPU__) || defined(__rtems__) || defined(__CYGWIN__) && !defined(__INSIDE_CYGWIN__)
+#if defined (__SPU__) || defined(__rtems__) || defined(__Unikraft__) || defined(__CYGWIN__) && !defined(__INSIDE_CYGWIN__)
 int    lstat (const char *__restrict __path, struct stat *__restrict __buf );
 int    mknod (const char *__path, mode_t __mode, dev_t __dev );
 #endif