--- a/newlib/libc/include/sys/signal.h  2022-10-23 15:34:20.636665821 +0200
+++ b/newlib/libc/include/sys/signal.h  2022-10-23 17:01:45.073247954 +0200
@@ -69,10 +69,27 @@
   int          si_signo;    /* Signal number */
   int          si_code;     /* Cause of the signal */
   union sigval si_value;    /* Signal value */
+
+  union {
+  struct {
+    void *si_addr;
+    short si_addr_lsb;
+      union {
+        struct {
+          void *si_lower;
+          void *si_upper;
+        } __addr_bnd;
+        unsigned si_pkey;
+      } __first;
+    } __sigfault;
+  } __si_fields;
+
 } siginfo_t;
+
+#define si_addr    __si_fields.__sigfault.si_addr
 #endif /* defined(_POSIX_REALTIME_SIGNALS) || __POSIX_VISIBLE >= 199309 */
 
-#if defined(__rtems__)
+#if defined(__rtems__) || defined(__Unikraft__)
 
 /*  3.3.8 Synchronously Accept a Signal, P1003.1b-1993, p. 76 */
